clang/test/CXX/drs/cwg6xx.cpp

```diff
@@ -922,7 +922,7 @@ namespace cwg667 { // cwg667: 8
 
   struct B { ~B() = delete; };
   union C { B b; };
-  static_assert(!__is_trivially_destructible(C), "");
+  static_assert(!__is_trivially_destructible(C), ""); // cxx26-error {{failed}}
```

Comment:
I think I want this static to be duplicated and put under `#if __cplusplus`, because we're not actually interested in "static_assert has failed" diagnostic.